CODEOWNERS

```diff
@@ -69,3 +69,4 @@ security-center @GoogleCloudPlatform/gcp-security-command-center @GoogleCloudPla
 retail @GoogleCloudPlatform/cloud-retail-team @GoogleCloudPlatform/nodejs-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
 media @GoogleCloudPlatform/cloud-media-team @GoogleCloudPlatform/nodejs-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
 healthcare @GoogleCloudPlatform/healthcare-life-sciences @GoogleCloudPlatform/nodejs-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
+routeoptimization @GoogleCloudPlatform/geo-routeoptimization @GoogleCloudPlatform/nodejs-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
```

Comment:
Hi @bshi Please confirm team exists:
> Unknown owner on line 72: make sure the team @GoogleCloudPlatform/geo-routeoptimization exists, is publicly visible, and has write access to the repository
routeoptimization @GoogleCloudPlatform/geo-routeoptimization @GoogleCloudPlatform/nodejs-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
Original file line numb

Reply:
Confirmed, see https://github.com/GoogleCloudPlatform/java-docs-samples/pull/9367#discussion_r1649561948

Reply:
Thanks, I had just confirmed the team myself here: https://github.com/orgs/GoogleCloudPlatform/teams/geo-routeoptimization

Reply:
I've added the team to the repository with write access. This should allow you to work off a clone as well. Thanks!